worlds/tunic/__init__.py

```diff
@@ -83,6 +83,8 @@ class TunicWorld(World):
     shop_num: int = 1  # need to make it so that you can walk out of shops, but also that they aren't all connected
     er_regions: Dict[str, RegionInfo]  # absolutely needed so outlet regions work
 
+    item_link_checked_locs: List[Location]  # so we don't make hints for the same spot more than once
```

Comment:
Is there a reason this is a list and not a set?

`if link_loc not in self.item_link_checked_locs` would probably benefit from it being a set.

Reply:
Good call, yeah